src/gourmet/GourmetRecipeManager.py

```diff
@@ -1,5 +1,6 @@
 import os
 import os.path
+from pkgutil import get_data as _get_data
```

Comment:
Please do 
```suggestion
from pkgutil import get_data
```

I remember your point of hiding the imports on import, yet it's not quite idiomatic Python.

Reply:
I would disagree. It may not be common, but in addition to being specifically mentioned in [PEP 8](https://www.python.org/dev/peps/pep-0008/#public-and-internal-interfaces), it's used in Python's own standard library:
https://stackoverflow.com/questions/47138594/why-are-python-imports-renamed-with-leading-underscores
https://stackoverflow.com/questions/20757763/why-are-modules-imported-as-name-in-another-module

But besides just stylistic issues, there's an important pragmatic reason as underscored in PEP 8:

> Imported names should always be considered an implementation detail. Other modules must not rely on indirect access to such imported names unless they are an explicitly documented part of the containing module's API

Note that imported names become part of a module's symbol table just like any other name, so `from foo import *` or `from foo import get_data` will work identically as if `get_data` was defined in the module itself. The best way to enforce this is by prefixing them with an underscore. While `__all__` provides partial mitigation in cases of `import *`, it doesn't guard against explicit imports:
```python
from foo import get_data  # this must be ok?
```
```python
from foo import _get_data  # this looks like a problem
```
And as noted in the PEP:

>Even with `__all__` set appropriately, internal interfaces ... should still be prefixed with a single leading underscore.